Numeric/SpecFunctions/Internal.hs

```diff
@@ -72,11 +79,33 @@ erf = c_erf
 -- \end{aligned}
 -- \]
 erfc :: Double -> Double
-{-# INLINE erfc #-}
+#ifdef USE_SYSTEM_ERF
 erfc = c_erfc
+#else
+erfc x
+    | x < 0.0   = 2.0 - a
+    | otherwise = a
+  where
+    z = abs x
+    t = 1.0 / (0.5 * z + 1.0)
+    a1 = t * 0.17087277 + (-0.82215223)
+    a2 = t * a1 + 1.48851587
+    a3 = t * a2 + (-1.13520398)
+    a4 = t * a3 + 0.27886807
+    a5 = t * a4 + (-0.18628806)
+    a6 = t * a5 + 0.09678418
+    a7 = t * a6 + 0.37409196
+    a8 = t * a7 + 1.00002368
+    a9 = t * a8
```

Comment:
This looks like Horner's summation. It could be replaced with `evaluatePolynomialL` for better readability

Reply:
I'm not familiar with internals. This is more or less direct translation of numeric recipes version.

I'm not competent enough to refactor that beyond what's written.

Reply:
Don't bother then. I'll fix both comment myself